src/main/java/server/EchoServer.java

```diff
@@ -1,10 +1,30 @@
 package server;
 
+import java.io.*;
+import java.net.ServerSocket;
+
 public class EchoServer {
-    public static String walkingSkeletonTest(){
-        return "testing 123";
+
+    public static void main(String[] args) throws IOException {
+        int clientPort = (args.length > 0) ? Integer.parseInt(args[0]) : 4444;
+        ServerSocket serverSocket = new ServerSocket(clientPort);
```

Comment:
Subject to what others say - I'm ok with this as-is for this PR, but I'd like a future refactoring PR where the Socket class gets its own wrapper, instead of dealing with a Server Socket object directly.

Reply:
socket class has been established